lib/datasource/local/models/amount_view_model.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/datasource/remote/model/token_model.dart';
+
+class AmountViewModel {
+  final double? amount;
```

Comment:
Is possible the amount be null?

Reply:
i think it should be non nullable

Reply:
The previous functionality was that in many many places in the UX this value can be null, and if it is it shows "" instead.

So I copied this into this class, it's a view model, and it's nullable, showing "" when null

Making it non-nullable is an option but then will complicated the code wherever this is used, since in almost all places it is used, the value can be null.

Should have explained this a little better.

I think the current optimal implementation is to have it be nullable. 

Otherwise all the places it's used will have this code 

amountViewModel?.toString() ?? ""

That's not too bad actually... 

I will look into the impact. The main reason I made it nullable - after it starting out non-nullable - is that nearly all places in the code that show an amount can have a nullable amount.

Reply:
I changed this - now the data model amount can't be null, but the places where we have null, the object can be null. 

Makes more sense.